students/10th/songsuho/style/main.css

```
@@ -0,0 +1,343 @@
+nav {
+    justify-content: space-between;
+    display: flex;
+    align-items: center;
+    padding: 0 100px;
+    border-bottom: 1px solid #dbdbdb;
+}
+
+.logo {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.logo_mini {
+    padding-right: 7px;
+    border-right: 1px solid black;
+}
+
+.logo_bar {
+    padding-left: 8px;
+}
+
+.point {
+    border-radius: 5px;
+    height: 30px;
+    width: 300px;
+    text-align: center;
+    /* position: relative; */
+}
+
+/* .point-1 {
+    width: 40px;
+    height: 20px;
+    left: 530px;
+    position: absolute;
+} */
+
+.menu {
+    display: flex;
+    justify-content: space-between;
+    width: 120px;
+    height: 60px;
+    margin: 0 87.5px;
+    padding-top: 30px;
+}
+
+.padding_box {
+    padding-top: 30px;
+    margin: 0 87.5px;
+    justify-content: center;
+    display: flex;
```

```suggestion
    display: flex;
    justify-content: center;
    margin: 0 87.5px;
    padding-top: 30px;
```
이 내용도 마찬가지 내용입니다. 기왕이면 중요한 내용을 위로, 연관성 있는 속성들 끼리 묶어서 정리해주세요. 나중에 스타일을 이리저리 고치다가 속성을 중복 적용하는 등의 경우를 방지할 수 있습니다.

---

수정하였습니다.